src/viam/media/video.py

```diff
@@ -18,33 +28,40 @@ class RawImage(NamedTuple):
     mime_type: str
     """The mimetype of the image"""
 
-    width: int
-    """The width of the image"""
-
-    height: int
-    """The height of the image"""
-
     def close(self):
         """Close the image and release resources. For RawImage, this is a noop."""
         return
 
 
 class CameraMimeType(str, Enum):
-    RAW = "image/vnd.viam.rgba"
+    VIAM_RGBA = "image/vnd.viam.rgba"
     JPEG = "image/jpeg"
     PNG = "image/png"
     PCD = "pointcloud/pcd"
+    UNSUPPORTED = "unsupported"
+
+    @classmethod
+    def from_lazy(cls, value: str) -> Tuple[Self, bool]:
+        is_lazy = False
+        mime_type = value
+        if value.endswith(LAZY_SUFFIX):
+            mime_type = value[: (len(value) - len(LAZY_SUFFIX))]
+            is_lazy = True
+        if not cls.is_supported(value) and not is_lazy:
+            mime_type = CameraMimeType(CameraMimeType.UNSUPPORTED)
+        return (cls(mime_type), is_lazy)
+
+    @property
+    def with_lazy_suffix(self) -> str:
+        return f"{self.value}{LAZY_SUFFIX}"
```

Comment:
[q] is there a chance that value might already the lazy suffix on it?

Reply:
No -- the only values allowed are the ones defined above. These enums are not extendible, so there's no risk of that